idseq_pipeline/commands/blacklist_functions.py

```diff
@@ -6,8 +6,8 @@
 from .common import *
 
 # arguments from environment variables
-INPUT_FASTA_S3 = os.environ.get('INPUT_FASTA_S3')
-ACCESSION2TAXID_DB_S3_PATH =  os.environ.get('ACCESSION2TAXID_DB_S3_PATH')
+INPUT_FASTA_S3 = get_env_or_err('INPUT_FASTA_S3')
```

Comment:
FYI 
```
INPUT_FASTA_S3 = os.environ['INPUT_FASTA_S3']
```
would also have the effect of getting the value or error otherwise -- but what you're doing, controlling the error message, is nicer